Add In Development stage workflow (#2)
sourcelevel/guidelines: resources/git_workflow.md

All story cards are prioritized based on importance vs effort to accomplish it.

### In Development

> When the card is in this stage and due extern facts can't be moved to Ready For Review stage, add `blocked` label

Reviewer: ```suggestion
When the User Story is in this stage and due some reason it can't be moved to Ready For Review stage, add `blocked` label
```

Author: Fixed in 886a8dd

> to the card, and leave it in this stage

Reviewer: ```suggestion
to the card leaving it in this stage. Also, comment on the User Story card explaining why it got blocked,
```

Author: Fixed in 886a8dd
